src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -736,6 +740,167 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                 .EndControl());
         }
 
+        private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Automation_GetError()
+        {
+            yield return new FormatViewDefinition("GetErrorInstance",
+                CustomControl.Create()
+                    .GroupByProperty("PSErrorIdentifier", label: "ErrorIdentifier")
+                    .StartEntry()
+                        .AddScriptBlockExpressionBinding(@"
+                            $maxDepth = 10
+                            $ellipsis = ""`u{2026}""
+                            $resetColor = ''
+                            if ($Host.UI.SupportsVirtualTerminal) {
+                                $resetColor = ""`e[0m""
+                            }
+
+                            function Get-VT100Color([string] $color) {
+                                if (! $Host.UI.SupportsVirtualTerminal) {
+                                    return ''
+                                }
+
+                                $colors = @{
+                                    'Black' = ""`e[2;30m""
+                                    'DarkRed' = ""`e[2;31m""
+                                    'DarkGreen' = ""`e[2;32m""
+                                    'DarkYellow' = ""`e[2;33m""
+                                    'DarkBlue' = ""`e[2;34m""
+                                    'DarkMagenta' = ""`e[2;35m""
+                                    'DarkCyan' = ""`e[2;36m""
+                                    'Gray' = ""`e[2;37m""
+                                    'DarkGray' = ""`e[1;30m""
```

Comment:
on non-Windows I believe that the `1` in `{esc}[1;30m` is _bold_ not _bright_ - this may be a difference between windows and non-Windows. Not sure what to do about it.

Reply:
Perhaps technically correct, but not sure what can be done either.  Conceptually, I think we can consider them equivalent.